protocol/bc/value.go

```diff
@@ -3,7 +3,17 @@ package bc
 type ValueSource struct {
 	Ref      Hash
 	Value    AssetAmount
-	Position uint64 // what int do we actually want?
+	Position uint64
+
+	// The Entry corresponding to Ref, if available
+	// The struct tag excludes the field from hashing
+	Entry `entry:"-"`
```

Comment:
Really this comment belongs on ValueDestination.Entry but that line isn't commentable.

Should ValueDestination.Entry have the `entry:"="` tag? It doesn't seem like we ever hash ValueDestination objects.